docs/installation.md

```diff
@@ -1,196 +1,79 @@
-
-Use of a virtual environment to contain all requirements is highly recommended:
-
-```bash
-virtualenv venv
-source venv/bin/activate
-pip install -r requirements.txt -r requirements-dev.txt
-
-# Installs metamist as a package
-pip install --editable .
-```
+It is recommended to use [pyenv](https://github.com/pyenv/pyenv) and [virtualenv](https://virtualenv.pypa.io/en/latest/installation.html) to manage the python version and virtual environment for metamist. Metamist currently is deployed with python 3.11 so it is best to use that locally to ensure your changes are compatible.
 
-You will also need to set the following environment variables. Adjust the paths if you installed the dependencies using an alternate means:
 
 ```bash
-# homebrew should export this on an M1 Mac
-# the intel default is /usr/local
-export HB_PREFIX=${HOMEBREW_PREFIX-/usr/local}
+# install python 3.11
+pyenv install 3.11
 
-# installing Java through brew recommendation
-export CPPFLAGS="-I$HB_PREFIX/opt/openjdk/include"
-export PATH="$HB_PREFIX/bin:$PATH:$HB_PREFIX/opt/openjdk/bin"
+# select python 3.11 for your current shell
+pyenv shell 3.11
 
-# installing liquibase through brew recommendation
-export LIQUIBASE_HOME=$(brew --prefix)/opt/liquibase/libexec
+# set up virtualenv with python 3.11
+virtualenv .venv -p 3.11
 
-# mariadb
-export PATH="$HB_PREFIX/opt/mariadb@11.7.2/bin:$PATH"
+# activate virtualenv
+source .venv/bin/activate
 
-# metamist config
-export SM_ENVIRONMENT=LOCAL # good default to have
-export SM_DEV_DB_USER=sm_api # makes it easier to copy liquibase update command
-
-```
-
-You can also add these to your shell config file e.g `.zshrc` or `.bashrc` for persistence to new sessions.
-
-#### PNPM/NVM Config
-
-Depending on your choice for using `pnpm` or `nvm` you will have to configure your shell for it.
-
-If you installed `pnpm`, you should have a similar snippet from the brew installation output:
-
-```shell
-export PNPM_HOME="/Users/$(whoami)/Library/pnpm"
-case ":$PATH:" in
-  *":$PNPM_HOME:"*) ;;
-  *) export PATH="$PNPM_HOME:$PATH" ;;
-esac
+# install requirements
+pip install --no-deps -r requirements-dev.txt
 ```
 
-Add this to your `.zshrc` to auto-load on next shell session.
-
-If you installed `nvm`, you will need to add lazy load since `nvm` has high load penalties.
+### Database setup
 
-- For Oh-My-Zsh users, you can just add the `nvm` plugin to your `.zshrc` via these [instructions](https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/nvm/README.md)
+#### Running MariaDB
 
-- If you do NOT have Oh-My-Zsh, you can use this [plugin](https://github.com/undg/zsh-nvm-lazy-load):
+Metamist uses a MariaDB 11 database. Docker is probably the easiest way to run the metamist MariaDB database locally.
```

Comment:
```suggestion
Metamist uses a MariaDB 11 database. Docker is the easiest way to run the Metamist MariaDB database locally.
```
Nit, instructional rather than narrative.